### ISSUE TITLE
Max message length (0) exceeded - when typing only 2 chars.

### ISSUE THREAD

**Issue by Kravet** [GetStream/stream-chat-android#953]
**Describe the bug**
Sometimes when open the MessageFragment and start typing I see the following validation error on the input field: "Max message length(0) exceeded." Even though I typed 2 chars. It happens relatively often - every 6th-10th time.

My MessageFragment is:
```
class MessageFragment : Fragment() {
    
    // This property is only valid between onCreateView and onDestroyView
    private var _binding: MessageFragmentBinding? = null
    private val binding get() = _binding!!

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        _binding = MessageFragmentBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }

    private lateinit var viewModel: MessageViewModel
    private val chanelId: String
        get() = MessageFragmentArgs.fromBundle(requireArguments()).channelId

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        viewModel = ViewModelProvider(this).get(MessageViewModel::class.java)

        // Step 1 - Create 3 separate ViewModels for the views so it's easy to customize one of the components
        val factory = ChannelViewModelFactory(chanelId)
        val channelHeaderViewModel: ChannelHeaderViewModel by viewModels { factory }
        val messageListViewModel: MessageListViewModel by viewModels { factory }
        val messageInputViewModel: MessageInputViewModel by viewModels { factory }

        // Step 2 - Bind the view and ViewModels, they are loosely coupled so it's easy to customize
        channelHeaderViewModel.bindView(binding.channelHeaderView, this)
        messageListViewModel.bindView(binding.messageListView, this)
        messageInputViewModel.bindView(binding.messageInputView, this)

        // Step 3 - Let the message input know when we open a thread
        messageListViewModel.mode.observe(viewLifecycleOwner) { mode ->
            when (mode) {
                is MessageListViewModel.Mode.Thread -> messageInputViewModel.setActiveThread(mode.parentMessage)
                is MessageListViewModel.Mode.Normal -> messageInputViewModel.resetThread()
            }
        }

        // Step 4 - Let the message input know when we are editing a message
        binding.messageListView.setOnMessageEditHandler {
            messageInputViewModel.editMessage.postValue(it)
        }

        // Step 5 - Handle navigate up state
        messageListViewModel.state.observe(viewLifecycleOwner) { state ->
            if (state is MessageListViewModel.State.NavigateUp) {
                Navigation.findNavController(requireView())
                        .runCatching { popBackStack() }
            }
        }

        // Step 6 - Handle back button behaviour correctly when you're in a thread
        binding.channelHeaderView.onBackClick = {
            messageListViewModel.onEvent(MessageListViewModel.Event.BackButtonPressed)
        }
    }

}
```

**SDK version**
"com.github.getstream:stream-chat-android:4.4.5"

**Steps to reproduce:**
Open any channel and start typing.

**Device:**
Vendor and model: OnePlus 5t (android 10), Samsung Galaxy S3.

**Screenshot with the issue:**
![image](https://user-images.githubusercontent.com/7354769/101067295-0df2fd80-35a0-11eb-9437-cb766b663f74.png)

**Comment by samiuelson** [comment]
This issue is already fixed with the following PR: https://github.com/GetStream/stream-chat-android/pull/926. It will be included in the next version (4.4.6) and released soon. cc @JcMinarro

**Comment by JcMinarro** [comment]
Hello @Kravet 

As Sam said, we have it already fixed. We have released version 4.4.6 that includes the patch.
Thanks
